### DISCUSSION TITLE
Model di quiz NN

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#475] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/206681117-95ee5ff4-bebf-41a4-96f3-c88d1b5a517a.png)
![image](https://user-images.githubusercontent.com/114885734/206681326-6a41d3ee-3682-4fc3-8128-c1ddbd8f8de9.png)
![image](https://user-images.githubusercontent.com/114885734/206681540-293b450b-720a-46d4-94be-c488bad72f5e.png)
Halo, mau tanya soal quiz NN, screen shot di atas berturut2 adalah hasil run untuk model base, model bigger, dan model tuning yang saya kerjakan sesuai instruksi di quiz. Ini saya kerjakannya benar atau salah ya? Dari hasil ini, menurut saya semua model overfit? (karena val_accuracy semuanya jauh lebih rendah dari data trainnya).

**Comment by dwiknrd** [comment, marked as answer]
Hi Bu [19807311](https://github.com/19807311), Terima kasih sudah bertanya.

Untuk model yang dibuat seharusnya terdapat salah satu model yang memiliki performa bagus dan tidak overfitting. Terkait hal tersebut pastikan bahwa ibu sudah menjalankan kode untuk mengunci ke randoman dari inisiasi bobot diawal, kemudian silahkan jalankan ulang untuk membuat arsitektur, compiler serta proses fittingnya.

Semoga bermanfaat, cheers :)
